apps/app/src/views/routes/Settings/components/users/UserItemActions.tsx

```diff
@@ -48,8 +55,21 @@ interface UserItemActionsProps {
   user: FragmentType<typeof userFragment>;
 }
 
+const RemoveUserFromOrganizationMutation = graphql(/* GraphQL */ `
+  mutation RemoveUserFromOrganization($userId: ID!) {
```

Comment:
```suggestion
  mutation UserItemActionRemoveUserFromOrganization($userId: ID!) {
```

Reply:
We should generally start naming mutations and fragments based on where they are being used in addition to what they do. This is inspired by the much stronger naming rule enforced by Relay. The point is that multiple places in code should be able to call `removeUserFromOrganization`, so either individual spot its used should be identifiable (and independently cached, but thats more for fragments)